pony-kafka/customnet/custom_tcp_connection_handler.pony

```diff
@@ -731,7 +733,11 @@ class CustomTCPConnectionHandler is TCPConnectionHandler
     ifdef windows then
       _close()
     else
-      _hard_close()
+      if _muted then
```

Comment:
This will break shutdown on dispose. It was implemented recently (https://github.com/WallarooLabs/pony-kafka/commit/9a863aabb2c708589d130f952d9766f12afe3065) to support wallaroo graceful shutdown requests.

Reply:
Ok, I'll fix this.